### ISSUE TITLE
[Accessibility]A11y__AzureDataStudio_NewJobSteps_Keyboard: Keyboard focus is not visible clearly on the "Close" and "Copy" buttons in the successful parsed message.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/azuredatastudio#5749]
**Bug Description:**
[“Check out Keros!](https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”
 GitHubTags: #A11yMAS; #A11yTCS; #MAS2.4.7; #A11y_AzureDataStudio; #Win32; #DesktopApp
**Environment Details:**
Application name: Azure Data Studio
Version: 1.8.0-insider (user setup)
Commit: 3e8f25d
Date: 2019-05-27T07:46:13.508Z
Electron: 3.1.8
Node.js: 10.2.0
OS: Windows_NT x64 10.0.17763

**Additional details:**
Tool name: Keyboard
MAS Number: MAS2.4.7

**Steps to Reproduce:**
1.Launch the Azure Data Studio application.
2.Enable the SQL server agent extension.
3.Navigate to the SQL agent tab from Home page->Click on new job button->fill the name field->navigate to the Steps tab.
4.Naviagte to the bottom of the pop-up and click on the "New Step" button.
5."New job step" screen opens.
6.Fill all the mandatory fields and hit enter on the parse button.
7.Press the shift + Tab key to navigate to the "Successfully parsed message" "copy" and "Close" button.
8. Observe if the keyboard focus is clearly visible on the "Copy" and "Close" button.

**Actual:**
Keyboard focus is not visible clearly on the "Close" and "Copy" buttons in the successful parsed message.

**Expected:**
Keyboard focus should be visible clearly on the "Close" and "Copy" buttons in the successful parsed message.

**User Impact:**
keyboard-only users will not understand where is the focus and face difficulty while traversing through the page.

**Recommendation:**
Please refer the following snippet link for fixes:
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

**MAS Reference:**
**MAS2.4.7-**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={3b98b04f-4fab-4d3e-81af-a0906a34849c}

**Attachment for Reference**
[5749_[Accessibility]A11y__AzureDataStudio_NewJobSteps_Keyboard.pptx](https://github.com/microsoft/azuredatastudio/files/3237015/5749_.Accessibility.A11y__AzureDataStudio_NewJobSteps_Keyboard.pptx)

Does this issue occur when all extensions are disabled?: Yes
